sdk/core/ci.yml

```diff
@@ -32,6 +32,7 @@ stages:
     parameters:
       ServiceDirectory: core
       CtestRegex: azure-core.
+      Coverage: enabled
```

Comment:
I'd suggest having this enabled for one test leg for all pipelines instead of having them opt in.

Reply:
Explained on the top. 
I want to explicitly disable what we know is not really or would fail, instead of letting it do nothing on the steps